docs/lite.md

```diff
@@ -17,13 +17,13 @@ The Pluggable Modular Parsers provide the ability to customize and extend the da
 The Splunk platform is the destination for the syslog data. Splunk allows for comprehensive analysis, searching, and visualization of the processed data.
 
 ##  Data Flow
-Source systems send syslog data to SC4S Lite. The data may be transmitted via UDP, TCP, or RELP, depending on the system's capabilities and configurations.
+Source systems send syslog data to SC4S Lite. The data may be transmitted via UDP or TCP depending on the system's capabilities and configurations.
 SC4S Lite receives the syslog data and routes it through the appropriate parsers, as defined by user configurations.
```

Comment:
SC4S Lite receives the syslog data and routes it through the parsers defined by your configurations. These parsers in the plugin module process the data, includig parsing, filtering, and enriching the data with metadata.

SC4S Lite forwards the processed syslog data to the Splunk platform over the HTTP Event Collector (HEC).